web3.md

```diff
@@ -824,11 +831,48 @@ loadCallSettings
     rule <k> #prepareTx TXID:Int
           => #clearLogs
           ~> #loadCallSettings TXID
-          ~> #executeTx TXID
-          ~> #makeTxReceipt TXID
+          ~> #validateTx TXID
          ...
          </k>
 
+    syntax KItem ::= "#validateTx" Int
+ // ----------------------------------
+    rule <k> #validateTx TXID => . ... </k>
+         <statusCode> ( _ => EVMC_OUT_OF_GAS) </statusCode>
+         <schedule> SCHED </schedule>
+         <message>
+           <msgID>      TXID     </msgID>
+           <txGasLimit> GLIMIT   </txGasLimit>
+           <data>       DATA     </data>
+           ...
+         </message>
+      requires ( GLIMIT -Int G0(SCHED, DATA, false) ) <Int 0
+
+    rule <k> #validateTx TXID => #executeTx TXID ~> #makeTxReceipt TXID ... </k>
+         <schedule> SCHED </schedule>
+         <callGas> _ => GLIMIT -Int G0(SCHED, CODE, true) </callGas>
```

Comment:
Can we make this one rule instead of two by putting `ACCOUNT_TO ==K .Account` as the third argument to `G0`